src/super_gradients/training/transforms/keypoint_transforms.py

```diff
@@ -71,6 +71,14 @@ def get_equivalent_preprocessing(self) -> List:
             preprocessing += t.get_equivalent_preprocessing()
         return preprocessing
 
+    def __repr__(self):
+        format_string = self.__class__.__name__ + "("
+        for t in self.transforms:
+            format_string += "\n"
+            format_string += "    {0}".format(repr(t))
```

Comment:
Just curious, why not directly `format_string += f"    {t}"` ?

Reply:
I just hit TAB to accept tabnine suggestion 🙈 
But you're right, this can be done better

Reply:
Fixed 99fd7f6d983b4f151b17c39feb78c704ab89f66d